monkey/infection_monkey/exploit/sshexec.py

```diff
@@ -86,12 +88,22 @@ def exploit_with_ssh_keys(self, port) -> paramiko.SSHClient:
                 )
                 self.add_vuln_port(port)
                 self.exploit_result.exploitation_success = True
+                self.publish_exploitation_event(
```

Comment:
It may be worthwhile to wrap `publish_exploitation_event()` and `report_login_attempt()` in a method, since they are called together in a few places

Reply:
Lets keep them separate for now as the login attempts may be a different event.